src/firebase/auth/functions/index.js

```diff
@@ -99,6 +145,12 @@ async function checkIfUserExists(data) {
   }
 }
 
+exports.onUserSignUp = functions.auth.user().onCreate(async (user) => {
+  if (user.email != null) {
+    return await produceMessage(user);
+  }
+});
```

Comment:
This function calls the kafka-produce-message function each time a user is created @Baalmart 
CC: @NoahNsimbe

Reply:
Alright @BenjaminSsempala , thanks!

A quick one, could we use the strict comparison operator "!==", as it ensures that both the value and the data type are the same?

In summary:
```
1 !== '1'   // returns true because the values are not strictly equal
0 !== false // returns true because the values are not strictly equal
```